config/enemies.py

```diff
@@ -0,0 +1,65 @@
+"""Enemies configuration file"""
+from enum import Enum
+
+class Ability(Enum):
+    MASKED = "Masked"
+    SPEEDER = "Speeder"
+    BOSS = "Boss"
+    BRUISER = "Bruiser"
+
+class Casual:
+    """casual enemy"""
+    ABLITY = None
+    HEALTH = 1
+    SPEED = 0.5
+
+    HEALTH_PER_LEVEL = 2
+    SPEED_PER_LEVEL = 0.25
+
+    IMAGE = "casual.png"
```

Comment:
nedavej za ten nazev tu koncovku .png, clovek, co bude hledat ty naloadovany textury prece nebude ve slovniku hledat pod klicem "casual.png", ale "casual". Navic, co kdyz se rozhodneme nepouzivat PNG, to to budeme pokazde prepisovat?